src/node_crypto.cc

```diff
@@ -3684,8 +3684,8 @@ void Hmac::HmacInit(const FunctionCallbackInfo<Value>& args) {
 bool Hmac::HmacUpdate(const char* data, int len) {
   if (!initialised_)
     return false;
-  HMAC_Update(&ctx_, reinterpret_cast<const unsigned char*>(data), len);
-  return true;
+  int r = HMAC_Update(&ctx_, reinterpret_cast<const unsigned char*>(data), len);
+  return r == 1;
```

Comment:
Looks like it can only occur with engines: https://github.com/openssl/openssl/commit/87d52468aa600e023, so untestable.